File: bigdata/aws/glue/etl_process/src/local_ubuntu.py
from com.spc.process.impl.etl_process import ETLProcess
from com.spc.spark.config.config_enum import ConfigEnum

def local():
    """
    Realiza as configurações para a execução do processo na máquina local
    """
    ETLProcess(ConfigEnum.LOCAL_UBUNTU.value).main_process()


local()

File: bigdata/aws/glue/scala_lab/src/main/scala/LocalUbuntu.scala
import com.spc.process.IProcess
import com.spc.process.impl.MainProcess
import com.spc.spark.config.ConfigEnum

object LocalUbuntu {
    def main(sysArgs: Array[String]): Unit = {
        val config = ConfigEnum.getInstance(ConfigEnum.LOCAL_UBUNTU)
        val spark = config.initSpark()
        val bucketName = config.getbucketName(sysArgs)
        val process: IProcess = new MainProcess()
        process.execute(spark, bucketName)
    }
}